pkg/services/sqlstore/migrations/playlist_mig.go

```diff
@@ -47,19 +57,15 @@ func addPlaylistUIDMigration(mg *Migrator) {
 		Postgres("UPDATE playlist SET uid=id::text;").
 		Mysql("UPDATE playlist SET uid=id;"))
 
-	mg.AddMigration("Add index for uid in playlist", NewAddIndexMigration(playlistV2(), &Index{
+	mg.AddMigration("Add index for uid in playlist", NewAddIndexMigration(playlistV2, &Index{
 		Cols: []string{"org_id", "uid"}, Type: UniqueIndex,
 	}))
-}
 
-func playlistV2() Table {
-	return Table{
-		Name: "playlist",
-		Columns: []*Column{
-			{Name: "id", Type: DB_BigInt, IsPrimaryKey: true, IsAutoIncrement: true},
-			{Name: "name", Type: DB_NVarchar, Length: 255, Nullable: false},
-			{Name: "interval", Type: DB_NVarchar, Length: 255, Nullable: false},
-			{Name: "org_id", Type: DB_BigInt, Nullable: false},
-		},
-	}
+	// Add columns used for kubernetes dual write synchronization
+	mg.AddMigration("Add playlist column created_at", NewAddColumnMigration(playlistV2, &Column{
+		Name: "created_at", Type: DB_BigInt, Nullable: false, Default: "0",
```

Comment:
Shouldn't this be `DATETIME` SQL type (same about `updated_at`)?

Reply:
with DATETIME, you can't apply a default to an existing structure :( so it can not be a required field, and that leads to needing pointers 🤮

Reply:
I'm not saying this as an argument in either direction, but I poked through the code and we're more likely to use an int64 than DATETIME in other tables, too 🤷🏻